pkg/cronjob/spark_dependencies_test.go

```diff
@@ -131,7 +170,9 @@ func TestSparkDependenciesResources(t *testing.T) {
 				Storage: v1.JaegerStorageSpec{
 					Type: "elasticsearch",
 					Dependencies: v1.JaegerDependenciesSpec{
-						Resources: dependencyResources,
```

Comment:
If we go for a backwards-compatible change, can you keep this test, adding a new case for the new property?

Reply:
It would only not be backward compatible for other Go apps (I guess operators) reusing the jaeger-types?